template/Data-Science/Classification/Mobile-User-Behavior-Classification/train.py

```
@@ -0,0 +1,134 @@
+        plt.tight_layout()
+        plt.show()
+    
+    def evaluate_model(self, model, X_test, y_test):
+        y_pred = np.argmax(model.predict(X_test), axis = 1) + 1
+        print(classification_report(y_test, y_pred))
+        print("Confusion Matrix:")
+        print(confusion_matrix(y_test, y_pred))
+
+    def save_model(self, model, model_path, scaler_path):
+        model.save(model_path)
+        file = open(scaler_path, 'wb')
```

Ruff (0.8.0)

Use a context manager for opening files

(SIM115)